@xen-orchestra/web-core/lib/components/UiChip.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <span class="ui-chip typo p3-regular" :class="[color, { disabled }]">
+    <span class="content" @click="emit('edit')">
+      <slot name="icon">
+        <UiIcon v-if="icon" :icon class="icon" fixed-width />
+      </slot>
+      <slot />
+    </span>
```

Comment:
```suggestion
    <UiIcon :icon class="icon" fixed-width />
    <span class="content">
      <slot />
    </span>
```

I removed the icon slot, because I don't think we'll ever need to customize the icon.

I also took it out of the label and removed the `v-if`, as this check is already done by the component itself.